context/requirements.txt

```diff
@@ -4,4 +4,6 @@ requests==2.22.0
 pyyaml==5.2
 python-datauri==0.2.8
 python-frontmatter==0.5.0
+vitessce==0.1.0a8
+pandas==1.1.2
```

Comment:
```suggestion
```
seems to be unused?

Reply:
https://github.com/vitessce/vitessce-python/issues/51